index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mint.com Clone</title>
+    <script
+      src="https://kit.fontawesome.com/ad0c4e912f.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="//fonts.googleapis.com/css?family=Cabin"
+    />
+  </head>
+  <body>
+    <nav>
+      <div>
+        <span>Already have an Intuit account?</span>
+        <a href="#"><span>I forgot my User ID or Password</span></a>
+      </div>
+      <button>
+        <span class="fa fa-unlock-alt"></span>
+        <span>Sign In</span>
+      </button>
+    </nav>
+
+    <main>
+      <section>
+        <div class="sec-top">
+          <div class="logo"></div>
+          <div class="icons">
```

Comment:
Here you have found a way around and used screenshots instead of icons. You should use icons with text.

Reply:
Hey Alex, but this is not a screenshot. We used Sprites to solve this. Isn't it allowed? I thought it was a valid method.

Reply:
I mean for better explanation it should have product images with text next to each product image
You can see from the screenshot the image and text are two different elements, hope I was clear enough
![image](https://user-images.githubusercontent.com/3244654/81108284-c4aed280-8f18-11ea-8ce9-e1c61051f525.png)

Reply:
Ok Alex, thanks for the clarification.